ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -205,4 +192,17 @@ interface {{ vlan_interface }}
 {%     if vlan_interfaces[vlan_interface].eos_cli is arista.avd.defined %}
```

Comment:
eos_cli should come last, since it can contain _any_ config, and the user must be able to override any of the generated config.